skimage/data/__init__.py

```diff
@@ -384,3 +385,33 @@ def stereo_motorcycle():
     return (load("motorcycle_left.png"),
             load("motorcycle_right.png"),
             np.load(_os.path.join(data_dir, "motorcycle_disp.npz"))["arr_0"])
+
+
+def hdr_images():
+    """
+    Bracket images useful for High Dynamic Range (HDR) imaging.
+
+    Images taken from the UNIS building on Svalbard, overlooking the
+    Hjorthfjellet mountain.
+
+    Returns
+    -------
+    ims : list
+        List containing three images.
+    exp : list
+        List of exposure times in seconds.
+
+    Notes
+    -----
+    These images were taken by Pål Ellingsen and are released into the public
+    domain (CC-0).
+    """
+
+    files = ['UNIS_EV-2.jpg',
```

Comment:
I've implemented this, but it doesn't seem to do anything more than delay the loading to later in the run. I've also added the option for using an ImageCollection in make_hdr, though I can't see any advantages of using one, as the order exposures are given as inputs needs to be the same